client/src/components/FlavorThumbnail.tsx

```diff
@@ -1,25 +1,25 @@
 import styles from "./flavor-thumbnail.module.css";
 
 export type Flavor = {
-  name: string,
-  price: number,
-  image: string,
+  name: string;
+  price: number;
+  image: string;
 };
 
-
-export default function FlavorThumbnail( {flavor} : {flavor: Flavor}) {
+export default function FlavorThumbnail({ flavor }: { flavor: Flavor }) {
   return (
-    <>    
-    <div className={styles.container}>
-      <img
-        className={styles.image}
-        src={flavor.image}
-        alt={`Photo of ${flavor.name}`}
-      />
-      <p className={styles.text}><strong>{flavor.name}</strong></p>
-      <p className={styles.text}>${flavor.price}</p>
-    </div>
+    <>
+      <div className={styles.container}>
+        <img
+          className={styles.image}
+          src={flavor.image}
+          alt={`Photo of ${flavor.name}`}
```

Comment:
Yay alt text! Could you add the word "ice cream" in there, unless that'll be in the name?